D/DataFrames/Compat.toml

```diff
@@ -50,24 +50,24 @@ Tables = "0.1.15-0"
 julia = "1"
 
 ["0.18-0.18.1"]
-PooledArrays = "0.5-0"
+PooledArrays = "0.5"
 
 ["0.18.2"]
 TableTraits = "0.4.0-1"
 Tables = "0.1.15-0"
 
 ["0.18.2-0.18"]
-CategoricalArrays = "0.5.2-*"
-Compat = "0.59.0-*"
+CategoricalArrays = "0.5.2-0.7"
+Compat = "0.59.0-2"
 
 ["0.18.2-0.19"]
-PooledArrays = "0.5.0-*"
+PooledArrays = "0.5.0"
```

Comment:
For future reference, this should be `"0.5"`, otherwise it excludes e.g. 0.5.1 etc.

Reply:
https://github.com/JuliaRegistries/General/pull/10521

Reply:
Thank you - I was not sure about that when I made a PR.